packages/ui/src/lib/checkbox/Checkbox.svelte

```diff
@@ -1,33 +1,53 @@
 <script lang="ts">
 import { faSquare as faOutlineSquare } from '@fortawesome/free-regular-svg-icons';
 import { faCheckSquare, faMinusSquare, faSquare } from '@fortawesome/free-solid-svg-icons';
-import { createEventDispatcher } from 'svelte';
+import { createEventDispatcher, type Snippet } from 'svelte';
 import Fa from 'svelte-fa';
 
-export let checked = false;
-export let disabled = false;
-export let indeterminate = false;
-export let disabledTooltip = '';
-export let title = '';
-export let id: string | undefined = undefined;
-export let name: string | undefined = undefined;
-export let required = false;
-
 const dispatch = createEventDispatcher<{ click: boolean }>();
+
+interface Props {
+  onclick?: (checked: boolean) => void;
+  children?: Snippet;
+  class?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  indeterminate?: boolean;
+  disabledTooltip?: string;
+  title?: string;
+  id?: string;
+  name?: string;
+  required?: boolean;
+}
+
+let {
+  onclick,
+  children,
+  class: className = '',
+  checked = $bindable(),
+  disabled = false,
+  indeterminate = false,
+  disabledTooltip = '',
+  title = '',
+  id = undefined,
```

Comment:
FYI you shouldn't need these `= undefined` since they are optional props above, but that's fine.